README.md

```diff
@@ -56,15 +56,25 @@ releases of the toolkit.
     * Python 3.6.2, along with a set of additional Python packages, is required to run some tools and workflows.
       GATK uses the [Conda](https://conda.io/docs/index.html) package manager to establish and manage the
       environment and dependencies required by these tools. The GATK Docker image comes with this environment
-      pre-configured. In order to establish an environment suitable to run these tools outside of the Docker image, the
-      conda [gatkcondaenv.yml](https://github.com/broadinstitute/gatk/blob/master/scripts/gatkcondaenv.yml) file is
-      provided. To establish the conda environment locally, [Conda](https://conda.io/docs/index.html) must first
-      be installed. Then, create the gatk environment by running the command ```conda env create -n gatk -f gatkcondaenv.yml```
-      (developers should run ```./gradlew createPythonPackageArchive```, followed by
-      ```conda env create -n gatk -f scripts/gatkcondaenv.yml``` from within the root of the repository clone).
-      To activate the environment once it has been created, run the command ```source activate gatk```. See the
-      [Conda](https://conda.io/docs/user-guide/tasks/manage-environments.html) documentation for
-      additional information about using and managing Conda environments.
+      pre-configured. In order to establish an environment suitable to run these tools outside of the Docker image,
+      do the following:
+        * Make sure [Conda or MiniConda](https://conda.io/docs/index.html) is installed. Miniconda is sufficient.
+        * If running from a zip or tar distribution:
```

Comment:
Instead of having three sets of instructions, perhaps separate these out into a section titled something like "first generate the conda environment files" with the three subsections, then another bullet with ```source activate gatk```, as its a little confusing to have to refer above when following the instructions, where above do you mean? etc

Reply:
Yes, its getting to be super confusing, especially since I had to add something that mentions the intel yml environment. Done.